schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/aep-hygiene-ops",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AEP Hygiene Operation Request",
+  "description": "AEP Hygiene operation request that instructs AEP to deletes or modify records in the specified dataset and sandbox",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "definitions": {
+    "hygieneOps": {
+      "properties": {
+        "xdm:batchTime": {
+          "title": "Time the batch was created",
+          "description": "Time the hygiene batch was created",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:operationType": {
+          "title": "Name of the hygiene operation to be applied by downstream consumers",
+          "description": "Name of the hygiene operation to be applied by downstream consumers.",
+          "type": "string",
+          "enum": ["identityDelete", "fieldUpdate"]
+        },
+        "xdm:targetDatasetId": {
+          "title": "Target Dataset ID specified by the user to apply hygiene operations",
+          "description": "May be \"*\", but only if operationType = \"identityDelete\". Otherwise a valid value is required.",
```

Comment:
The "identityDelete" part of this should be changed to "delete_identity".

Reply:
Done